tests/app/test_event_handlers.py

```diff
@@ -1,6 +1,7 @@
 import uuid
 from unittest.mock import ANY
```

Comment:
We'll need to add an import here for `patch`:

```suggestion
from unittest.mock import ANY, patch
```